### PR TITLE
update example project to use sdk plus 2.0.0-rc1

### PR DESCRIPTION
**Pull request by tzahie**

### CODE REVIEW

File: Examples/TaboolaSampleNotificaions/app/src/main/java/notifications/samples/taboola/com/taboolasamplenotificaions/SampleApplication.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        super.onCreate();
+        Logger.setLogLevel(Logger.DEBUG);
+        initSdkPlus(this);
+    }
+
+    private void initSdkPlus(Context appContext) {
+        SdkPlusPublisherInfo sdkPlusPublisherInfo = new SdkPlusPublisherInfo();
+        sdkPlusPublisherInfo.setConfigId("conf_dev");
+        sdkPlusPublisherInfo.setPublisherName("dt-newsplace-notification");
+        sdkPlusPublisherInfo.setActiveFeatures(SCHEDULED_NOTIFICATIONS);
```
Comment (orenktaboola) [inline Examples/TaboolaSampleNotificaions/app/src/main/java/notifications/samples/taboola/com/taboolasamplenotificaions/SampleApplication.java:36]: is this an example just for scheduled notifications ?

File: Examples/TaboolaSampleNotificaions/app/src/main/java/notifications/samples/taboola/com/taboolasamplenotificaions/SampleApplication.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                final String text = plusFeature.name() + " init failed";
+                showToast(appContext, text);
+                // TODO: handle int failure in your code
+            }
+        };
+
+        TaboolaSdkPlus.init(sdkPlusPublisherInfo, sdkPlusInitCallback);
+    }
+
+    private void broadcast(final Context appContext, PlusFeature plusFeature) {
```
Comment (orenktaboola) [inline Examples/TaboolaSampleNotificaions/app/src/main/java/notifications/samples/taboola/com/taboolasamplenotificaions/SampleApplication.java:57]: add comment what is this function (and i am not sure just "broadcast" is a good function name)

File: Examples/TaboolaSampleNotificaions/app/src/main/java/notifications/samples/taboola/com/taboolasamplenotificaions/SettingActivity.java [hunk 3]
```diff
@@ -33,37 +41,30 @@ protected void onCreate(Bundle savedInstanceState) {
...
-
-                    setLastUsedSettings();
-
-                    initSettingsLayout();
-                },
-                throwable -> {
-                    Log.e(TAG, "initTaboolaSdkPLus: " + throwable.getMessage(), throwable);
-                    // todo handle init fail
-                });
+    private void onScheduledNotificationInitFinished() {
```
Comment (orenktaboola) [inline Examples/TaboolaSampleNotificaions/app/src/main/java/notifications/samples/taboola/com/taboolasamplenotificaions/SettingActivity.java:57]: again comments. if it's an example app don't assume the publisher understands the code

File: Examples/TaboolaSampleNotificaions/app/src/main/java/notifications/samples/taboola/com/taboolasamplenotificaions/SampleApplication.java (hunk 1 above)
Comment (tzahie) [reply Examples/TaboolaSampleNotificaions/app/src/main/java/notifications/samples/taboola/com/taboolasamplenotificaions/SampleApplication.java:36]: yes we have 3 examples 
1. simple only SCHEDULED
2. same + feed_api + feed_std
3. both Features (with push as well)

File: Examples/TaboolaSampleNotificaions/app/src/main/java/notifications/samples/taboola/com/taboolasamplenotificaions/SampleApplication.java (hunk 2 above)
Comment (tzahie) [reply Examples/TaboolaSampleNotificaions/app/src/main/java/notifications/samples/taboola/com/taboolasamplenotificaions/SampleApplication.java:57]: fixed.

File: Examples/TaboolaSampleNotificaions/app/src/main/java/notifications/samples/taboola/com/taboolasamplenotificaions/SettingActivity.java (hunk 3 above)
Comment (tzahie) [reply Examples/TaboolaSampleNotificaions/app/src/main/java/notifications/samples/taboola/com/taboolasamplenotificaions/SettingActivity.java:57]: fixed.

File: Examples/TaboolaSampleNotificaions/app/src/main/java/notifications/samples/taboola/com/taboolasamplenotificaions/SettingActivity.java [hunk 4]
```diff
@@ -116,4 +116,38 @@ private void initSettingsLayout() {
         recyclerViewCategories.setLayoutManager(new LinearLayoutManager(this));
         recyclerViewCategories.setAdapter(adapter);
     }
+
+
+    private BroadcastReceiver receiver = new BroadcastReceiver() {
```
Comment (orenktaboola) [inline Examples/TaboolaSampleNotificaions/app/src/main/java/notifications/samples/taboola/com/taboolasamplenotificaions/SettingActivity.java:121]: add comment what is this receiver for

**Pull request closed without merging**